src/main/scala/com/tapad/docker/DockerComposePlugin.scala

```diff
@@ -414,16 +415,23 @@ class DockerComposePluginLocal extends AutoPlugin with ComposeFile with DockerCo
     }
   }
 
-  def composeTestRunner(implicit state: State, args: Seq[String]): State = {
+  def composeTestRunner(state: State, args: Seq[String]): State = {
     val newState = getPersistedState(state)
 
     val requiresShutdown = getMatchingRunningInstance(newState, args).isEmpty
     val (preTestState, instance) = getTestPassInstance(newState, args)
 
-    val finalState = if (getSetting(testPassUseSpecs2))
-      runTestPassSpecs2(preTestState, args, instance)
-    else
-      runTestPass(preTestState, args, instance)
+    val cucumberState = if (getSetting(testPassUseCucumber)(preTestState))
```

Comment:
When trying to run locally I noticed that my Specs2 example was not finding the docker instance to shutdown after test case execution. When I put the `implicit` back and cleaned up the logic a bit it worked as expected.
```
  def composeTestRunner(implicit state: State, args: Seq[String]): State = {
    val newState = getPersistedState(state)

    val requiresShutdown = getMatchingRunningInstance(newState, args).isEmpty
    val (preTestState, instance) = getTestPassInstance(newState, args)

    val finalState = if (getSetting(testPassUseCucumber))
      runTestPassCucumber(preTestState, args, instance)
    else if (getSetting(testPassUseSpecs2)) {
      runTestPassSpecs2(preTestState, args, instance)
    } else {
      runTestPass(preTestState, args, instance)
    }

    if (requiresShutdown)
      stopDockerCompose(finalState, Seq(instance.get.instanceName))
    else
      finalState
  }
```